qt/widgets/instrumentview/inc/MantidQtWidgets/InstrumentView/UnwrappedSurface.h

```diff
@@ -62,7 +62,8 @@ namespace MantidWidgets {
 class EXPORT_OPT_MANTIDQT_INSTRUMENTVIEW UnwrappedSurface : public ProjectionSurface {
   Q_OBJECT
 public:
-  explicit UnwrappedSurface(const InstrumentActor *rootActor);
+  explicit UnwrappedSurface(const InstrumentActor *rootActor, const QSize &widgetSize,
+                            const bool correctAspectRatio = true);
```

Comment:
I think the name change may have been missed in header's constructors?

Reply:
No that's a mistake! I'll fix